File: Python P/Functions/recursive-function/fabbonacci.py
num = int(input('Enter a Number to calculate Fabbonacci Series'))
def fabb(n):
    if n == 0:
        return 0
    if n == 1:
        return 1
    else:
        return fabb(n-1) + fabb(n-2)
print('The fabbonacci series for the entered number is :')
for i in range(num):
    print(fabb(i),"\t",end="")

File: recursion/fabbonacci.c
#include<stdio.h>
int fab(int n){
    if(n==0){
        return 0;
    }
    if(n==1){
        return 1;
    }
    return fab(n-1) + fab(n-2);
}

int main(){
    int num,i;
    printf("Enter the number to calculate fabbonacci series\n");
    scanf("%d",&num);
    printf("The fabbonacci series is :");
    for(i=0;i<num;i++){
    printf("%d\t",fab(i));
    }
}